src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Adds a step to deploy artifacts on environments in xl-deploy server using jenkins.
```

Comment:
Adds a step to deploy artifacts on environments in the XL Deploy server using Jenkins.